build/__nodeVersions.sh

```diff
@@ -1,7 +1,7 @@
 # This file was auto-generated from 'constants.yaml'. Changes may be overridden.
 
-YARN_VERSION='1.13.0'
-YARN_MINOR_VERSION='1.13'
+YARN_VERSION='1.17.3'
+YARN_MINOR_VERSION='1.17'
```

Comment:
Hmm, this variable name got me confused...should it have said `YARN_MAJOR_MINOR_VERSION`?

Reply:
Can you kick off a Nightly too?

Reply:
sure